packages/mobile/e2e/scripts/utils.ts

```diff
@@ -0,0 +1,41 @@
+const Web3 = require('web3')
+const ContractKit = require('@celo/contractkit')
+const dotenv = require('dotenv')
+const web3 = new Web3('https://alfajores-forno.celo-testnet.org')
+const kit = ContractKit.newKitFromWeb3(web3)
+var consts = require('./consts.ts')
```

Comment:
AFAIK, using `require` we don't get compile time TS type checks.  Can we switch to `import`?

Reply:
I ended up exporting these constants and using imports in `check-e2e-wallet-balance.ts` and `fund-e2e-accounts.ts` as seen [here](https://github.com/valora-inc/wallet/blob/tomm/low-balance-fast-fail/packages/mobile/e2e/scripts/utils.ts#L2-L6).